pandas/tests/test_generic.py

```diff
@@ -1874,6 +1874,52 @@ def test_pipe_panel(self):
         with tm.assertRaises(ValueError):
             result = wp.pipe((f, 'y'), x=1, y=1)
 
+    def test_align_joins(self):
+        df = DataFrame(np.array([[1., 2.], [3., 4.]]), columns=list('AB'))
```

Comment:
add the issue number as a comment here

rename to ``test_align_broadcast_axis``